extensions/energy/src/test/groovy/org/openremote/extension/energy/ForecastSolarServiceTest.groovy

```diff
@@ -144,10 +142,8 @@ class ForecastSolarServiceTest extends Specification implements ManagerContainer
         def assetProcessingService = container.getService(AssetProcessingService.class)
         def forecastSolarService = container.getService(ForecastSolarService.class)
 
-        expect: "a Future for calculation should exist and the asset should have filled in value for power and powerForecast"
+        expect: "the asset should have filled in value for power and powerForecast"
         conditions.eventually {
```

Comment:
I don't know enough about the business logic underneath but why remove some of those tests ?